### PR TITLE
SISRP-19992 Correctly parse EDO DB locations without room numbers

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.berkeley.edu/browse/SISRP-19992

And un-pend the spec.

### CODE REVIEW

**Comment by redconfetti** [comment]
LGTM, @johncrossman ?

File: app/models/edo_oracle/course_sections.rb [hunk 1]
```diff
@@ -48,7 +48,12 @@ def translate_location(meeting)
       if meeting['location'] == 'Requested General Assignment'
         building_name = 'Room not yet assigned'
       else
-        building_name, room_number = meeting['location'].rpartition(/\s+/).reject &:blank?
+        # See spec for behavior of this regex.
+        if (partitioned = meeting['location'].match /\A(.*)\s+(\d[^\s]*)\Z/)
```
Comment (johncrossman) [inline app/models/edo_oracle/course_sections.rb]: @paulkerschen Not sure where we draw the line on edge cases but this pattern does not match locations such as `Hearst Field Annex C57`

Comment (paulkerschen) [inline app/models/edo_oracle/course_sections.rb]: @johncrossman I will amend and add that location to the spec. :)

**Comment by johncrossman** [comment]
Thanks, @paulkerschen  Waiting on T

**Comment by johncrossman** [comment]
All we need is one green Travis

**Pull request merged**